src/System.Private.CoreLib/shared/System/Guid.cs

```diff
@@ -939,23 +939,19 @@ public int CompareTo(Guid value)
             return 0;
         }
 
-        public static bool operator ==(Guid a, Guid b)
-        {
+        public static bool operator ==(Guid a, Guid b) =>
             // Now compare each of the elements
```

Comment:
nit: not related to your changes, but imho the existing comments do not add any value and should be deleted

https://github.com/dotnet/coreclr/blob/57361bc9b78c4d21fe43ff48c21083993715c0ab/src/System.Private.CoreLib/shared/System/Guid.cs#L789-L794

```suggestion
            // Now compare each of the elements
```